### DISCUSSION TITLE
how do i flash my bios

### DISCUSSION THREAD

**Discussion by Musa-Irfan** [xCuri0/ReBarUEFI#236] [Q&A]
i have my pacthed bios my mobo is xps 8500 but how am i supposed to flash it originally i used an exe file but i cant use that now

**Comment by AnthonyWatson** [comment]
It sounds like you're running a non-Windows OS, correct?

If so, will one of the options for Non-Windows users here work?

https://www.dell.com/support/home/en-us/drivers/driversdetails?driverid=gvgpd#importanttab
